features/step_definitions/search_steps.rb

```diff
@@ -19,6 +19,14 @@
   end
 end
 
+Given (/^the following engagements exist for "([^"]*)":$/) do |app_name, table|
+  @app = App.find_by_name(app_name)
+  table.hashes.each do |hash|
+		@app.engagements.create(hash)
+	end
+  @app.save!
+end
+
```

Comment:
Already included in previous PR, but to pass CI, we have to include new steps.

Please ignore this file.